migrate/migrations.go

```diff
@@ -439,16 +440,16 @@ func (m *Migrations) genMigrationName(name string) (string, error) {
 }
 
 func (m *Migrations) runUp(ctx context.Context, db *bun.DB, migration *Migration) error {
-	m.log.Printf("running migration %s... ", migration.Name)
+	m.printer.Printf("running migration %s... ", migration.Name)
 	if migration.Up != nil {
 		if err := migration.Up(ctx, db); err != nil {
-			m.log.Printf("unable to run migration %s: %s", migration.Name, err)
+			m.printer.Printf("failure: %s\n", err)
```

Comment:
No need to repeat the migration twice, because the full log string is `running migration %s... failure: %s\n`